### PR TITLE
[FX-2191] Implements installation shot carousel

### PR DESCRIPTION
**Pull request by dzucconi**
Closes: [FX-2191](https://artsyproduct.atlassian.net/browse/FX-2191)

Nothing fancy (yet) for the modal view, just displays a larger version of the image, backdropped.

### CODE REVIEW

File: src/v2/Apps/Show/components/ShowInstallShots.tsx [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+          >
+            <ResponsiveBox
+              maxWidth={selectedImage.zoom1x.width}
+              maxHeight={selectedImage.zoom1x.width}
+              aspectWidth={selectedImage.zoom1x.width}
+              aspectHeight={selectedImage.zoom1x.height}
+            >
+              <Image
+                src={selectedImage.zoom1x.src}
+                srcSet={`${selectedImage.zoom1x.src} 1x, ${selectedImage.zoom2x.src} 2x`}
```
Comment (anandaroop) [inline src/v2/Apps/Show/components/ShowInstallShots.tsx:74]: Noice. As a tangent where would I learn more about our support for `srcSet`? There's a Palette PR for `Image` — and some Gemini updates I presume?

File: src/v2/Apps/Show/components/ShowInstallShots.tsx [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+          }
+          _2x: resized(height: 400) {
+            src: url
+          }
+          zoom1x: resized(width: 900, height: 900) {
+            src: url
+            width
+            height
+          }
+          zoom2x: resized(width: 900, height: 900) {
```
Comment (anandaroop) [inline src/v2/Apps/Show/components/ShowInstallShots.tsx:113]: Are these meant to be resized to the same dimensions?

Comment (dzucconi) [reply src/v2/Apps/Show/components/ShowInstallShots.tsx:113]: OH

File: src/v2/Apps/Show/components/ShowInstallShots.tsx (hunk 1 above)
Comment (dzucconi) [reply src/v2/Apps/Show/components/ShowInstallShots.tsx:74]: We just pass through the native prop so just the usual suspects: https://developer.mozilla.org/en-US/docs/Learn/HTML/Multimedia_and_embedding/Responsive_images

Comment (dzucconi) [reply src/v2/Apps/Show/components/ShowInstallShots.tsx:74]: I'd like to handle this automatically in Metaphysics and just request `src` and `srcSet` attrs for a given size and have these filled out. But that's just been a TODO for me for a while.

File: src/v2/Apps/Show/components/ShowInstallShots.tsx (hunk 2 above)
Comment (dzucconi) [reply src/v2/Apps/Show/components/ShowInstallShots.tsx]: Thanks — resolved

File: src/v2/Apps/Show/components/ShowInstallShots.tsx [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+  {
+    show: graphql`
+      fragment ShowInstallShots_show on Show {
+        name
+        images {
+          internalID
+          mobile1x: resized(height: 300) {
+            width
+            height
+          }
```
Comment (dzucconi) [inline src/v2/Apps/Show/components/ShowInstallShots.tsx:99]: The spec calls for a 300 height limit on mobile so we're just using this to get the resized dimensions and using responsive props to adjust the image sizes at that breakpoint.

File: src/v2/Apps/Show/components/ShowInstallShots.tsx (hunk 2 above)
Comment (anandaroop) [reply src/v2/Apps/Show/components/ShowInstallShots.tsx]: 👍

**Review by anandaroop — APPROVED** [review]

**Pull request merged**